src/TaskList.js

```diff
@@ -1,14 +1,39 @@
 class TaskList {
-  static #taskList = [
-    { description: 'aaa', completed: true, index: 1 },
-    { description: 'bbb', completed: true, index: 2 },
-    { description: 'ccc', completed: false, index: 3 },
-    { description: 'ddd', completed: false, index: 4 },
-  ];
-
-  static getTaskList() {
-    return this.#taskList;
+  static #taskList = [];
+
+  static #lsTaskList = 'taskList';
+
+  static getTaskList = () => TaskList.#taskList;
+
+  static setStore() {
+    localStorage.setItem(TaskList.#lsTaskList, JSON.stringify(TaskList.#taskList));
+  }
+
+  static getStore() {
+    TaskList.#taskList = JSON.parse(localStorage.getItem(TaskList.#lsTaskList));
   }
+
+  static add = (task) => {
+    const index = TaskList.#taskList.length + 1;
+    const newTask = { ...task, completed: false, index };
+    TaskList.#taskList.push(newTask);
+    TaskList.setStore();
+    return newTask;
+  };
+
+  static get = (index) => TaskList.#taskList[index - 1];
+
+  static modify = (task) => {
+    const { index } = task;
+    TaskList.#taskList[index - 1] = task;
+    TaskList.setStore();
+  };
+
+  static remove = (index) => {
+    const list = TaskList.#taskList.filter((task) => task.index !== index);
+    TaskList.#taskList = list.map((task, i) => ({ ...task, index: i + 1 }));
+    TaskList.setStore();
+  };
```

Comment:
- [ ] I should be able to remove a task but Because I am not able to add this is not possible please address that